source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRViewController.mm

```diff
@@ -241,11 +241,13 @@ - (void) addTasksToConcurrentQueue:(std::vector<std::shared_ptr<BaseCardElement>
                         // Convert html string to NSMutableAttributedString, NSAttributedString knows how to apply html tags
                         NSData *htmlData = [parsedString dataUsingEncoding:NSUTF16StringEncoding];
                         NSDictionary *options = @{NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType};
-                        // Initializing NSMutableAttributedString for HTML rendering is very slow
-                        NSMutableAttributedString *content = [[NSMutableAttributedString alloc] initWithData:htmlData options:options documentAttributes:nil error:nil];
 
-                         dispatch_async(dispatch_get_main_queue(),
```

Comment:
I have questions, Are there any impacts on performance by doing this? and is there any co-relation with rendering completion notification changes? 
Thanks

Reply:
A few from us tested the change and did not see any noticeable degradation.

There is no co-relation with rendering completion notification. We got call stack at the crash and see it's from HTML rendering.

Reply:
great;

thanks!

---
In reply to: [171457808](https://github.com/Microsoft/AdaptiveCards/pull/1196#discussion_r171457808) [](ancestors = 171457808)